app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupViewModel.kt

```diff
@@ -23,8 +23,8 @@ class InviteGroupViewModel @Inject constructor(
 
         val filterFriends = liveData<List<User>> {
                 val result = MediatorLiveData<List<User>>()
-                result.addSource(friends) { friendList ->
-                        result.value = getFilterFriends(friendList ?: emptyList(), textSearch.value ?: "")
+                result.addSource(friends) { _ ->
+                   //     result.value = getFilterFriends(friendList ?: emptyList(), textSearch.value ?: "")
```

Comment:
Why comment out? is ok?

Reply:
it ok ,  Show only when the user searches